theme/templates/resource-landing-page/title-section.html

```diff
@@ -102,6 +84,35 @@
     {% endif %}
 {% endif %}
 
+{# ======= Check MODFLOW Model Instance content files =======#}
+{% if cm.resource_type == "MODFLOWModelInstanceResource" %}
+    {% if cm.check_content_files %}
+        <div class="col-sm-12">
+            <div id="missing-metadata-or-file" class="alert persistent {% if just_created %}alert-success{% else %}alert-warning{% endif %} alert-dismissible" role="alert">
+                <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span
+                        aria-hidden="false">&times;</span></button>
+
+                    {% if cm.check_content_files == 'multiple_nam' %}
+                        <br>
+                        <span>The following condition(s) is(are) recommended to have a complete resource before making the resource public or discoverable:<br>
+                                <ul>
+                                   <li>Only one .nam or .mfn is allowed</li>
+                                </ul></span>
+                    {% else %}
+                        <br>
+                        <span>The following file(s) is(are) recommended to have a complete resource before making the resource public or discoverable:<br>
```

Comment:
I think this message is a little confusing to read. I think it's better to use plurals only. Even if there is only one item, the plurals should still make sense since we are referencing a list.
So, something like:

> The following files are recommended to have a complete resource before making the resource public or discoverable